README.md

```diff
@@ -78,6 +78,6 @@ Some things to keep in mind:
 
 **YOUR NOTES**
 ```
-
+I worked on this lab with a few classmates last night. I chimed in as much as i could but still struggling with the pace. Forms is really where I need a ton of reps and the practice excercises have been helping with that
```

Comment:
Keep on practicing and if you're not already, I recommend going back through the exercise by yourself from scratch to review the concepts you covered with the others. Let us know if you have any questions.